index.bs

```diff
@@ -111,6 +111,29 @@ and by name association.
 
 Please consult widely on names in your APIs.
 
+<h3 id="feature-detect">New features should be detectable</h3>
+
+The existence of new features should generally be detectable,
+so that web content can act appropriately if the feature is present or not.
+This applies both to features that
+are not present because they are not implemented,
+and to features that may not be present in a particular configuration
+(ranging from features that are present only if particular hardware is present,
```

Comment:
Perhaps "present only on particular platforms"? It would be strange if, for instance, the WebUSB API disappeared entirely when no USB devices were plugged in.